py/server/deephaven/table.py

```diff
@@ -2298,6 +2309,11 @@ def table(self) -> Table:
             self._table = Table(j_table=self.j_partitioned_table.table())
         return self._table
 
+    @property
+    def update_graph(self) -> _JUpdateGraph:
```

Comment:
Two problems:
1) The API is exposing Java objects to the user.
2) The type hint is referring to a private variable.